plugins/opencv/ocv_stereo_depth_map.cxx

```diff
@@ -126,13 +126,23 @@ ::compute( kwiver::vital::image_container_sptr left_image,
 {
   cv::Mat ocv1 = arrows::ocv::image_container::vital_to_ocv( left_image->get_image() );
   cv::Mat ocv2 = arrows::ocv::image_container::vital_to_ocv( right_image->get_image() );
+  cv::Mat ocv1_gray, ocv2_gray;
+
+  if (ocv1.channels() > 1) {
```

Comment:
Existing styles for brackets start on next line not the same line as the if statement (that we don't want to mix and match where necessary)

Reply:
Done.